psqlgraph/query.py

```diff
@@ -542,8 +541,8 @@ def has_sysan(self, keys):
         :param str key: System annotation key
 
         """
-        if isinstance(keys, str):
+        if isinstance(keys, six.string_types):
             keys = [keys]
         for key in keys:
-            self = self.filter(self.entity()._sysan.has_key(key))
+            self = self.filter(key in self.entity()._sysan)
```

Comment:
is this right?

Reply:
looks ok, `has_key` has been deprecated, its either this or `__contains__`